index.js

```diff
@@ -237,6 +237,28 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		if (signal === undefined || signal === 'SIGTERM') {
+			const retryAfter = options !== undefined &&
```

Comment:
Instead of an `options` object, wouldn't passing the argument directly as a number/string be simpler?

Reply:
Easier for sure. But I'd also say more confusing. The API for the normal [kill](https://nodejs.org/dist/latest-v10.x/docs/api/process.html#process_process_kill_pid_signal) has the signal and an integer as well - just reversed. So people might be thinking it's the same behavior.
I'm also a huge fan of options parameters in general since it let's you update your API without getting hated by your userbase.

Reply:
Yes I agree with you. Myself I usually prefer named parameters and options objects, because it's easier to extend as you mention. However, for parameters that are not expected to be extended too much (which is arguably hard to predict), simple types might be more developer-friendly I believe.

Either way works for me:
  1) the current solution: `options` object with `retry` boolean and `retryAfter` string/number
  2) `options` boolean/string/number

I would like to know what @sindresorhus thinks about it?